ow_plexil/src/plans/TestFaultDependencies.plp

```diff
@@ -0,0 +1,43 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "plan-interface.h"
+
+// Tests the FaultDependencies class. Depending on the XML file used, 
+// the fault info of a subsystem will be printed out.
+
+
+TestFaultDependencies:
```

Comment:
Please have this plan print a start message, so that it's more obvious that it's running.